lib/market.rb

```diff
@@ -0,0 +1,17 @@
+class FarMar::Market
+  attr_reader :market_id, :name, :address, :city, :state, :zip
+
+  def read_csv(file)
+    CSV.read(file, 'r')
```

Comment:
This is a pretty clever solution! Reading the csv each time is pretty expensive though; I think you'll feel your tests slowdown a bit as your start rummaging through the larger files. A possible refactor would be to read the csv once, at the beginning on `initialize`, and set it to a local variable, like `raw_data = read_csv("./support/markets.csv")`.

Once you've done that, you can assign your instance variables like you're already doing, but reference the `raw_data` variable instead, like `@name = raw_data[csv_index][2]`.